packages/orchestration/src/facade.js

```diff
@@ -42,6 +45,30 @@ const makeLocalChainFacade = localchain => {
           console.log('deposit got', payment);
           return E(account).deposit(payment);
         },
+        async getAddress() {
+          const addressStr = await E(account).getAddress();
+          return {
+            address: addressStr,
+            chainId: 'agoric-3',
+            addressEncoding: 'bech32',
+          };
+        },
+        getBalance(_denom) {
+          // FIXME map denom to Brand
+          const brand = /** @type {any} */ (null);
+          return E(account).getBalance(brand);
+        },
+        getBalances() {
+          throw new Error('not yet implemented');
+        },
+        send(toAccount, amount) {
+          // FIXME implement
+          console.log('send got', toAccount, amount);
+        },
```

Comment:
local-chain-account-kit implements `send()`, right? are we not ready to wire it up?

Reply:
To know it's covered I've been leaning to writing a breaking test first and then implementing until green